languages/go/exercises/concept/time-duration/.docs/instructions.md

```diff
@@ -0,0 +1,48 @@
+In this exercise you'll be working on an class room time schdule for a school. 
+
+You have five tasks, which will all involve scheduling of class periods and intervals.The class duration is 1 hr in all the tasks given blow.
+
+## 1. Parse the Duration of the Class
+
+Implement the `StartTime` function to parse a textual representation of a time duration of the class into the corresponding `time.Duration` format and print the time:
+
+```go
+StartTime("1h")
+// => 1h0m0s
+```
+
+## 2. Check if the class is over
+
+Implement the `IsOver` function that takes the start time and a given time and finds out whether the current class is over or not :
+
+```go
+IsOver("7/25/2019 13:45:00","July 25, 2019 14:46:00")
+// => true
+```
```

Comment:
I don't think we need duration for this one...

Reply:
`time.Parse` should not be part of this exercise any more. That has been done in the `time` exercise before.